networking/change-block-size.md

```diff
@@ -19,9 +19,67 @@ This how-to guide uses the following {{site.prodname}} features:
 
 ### Concepts
 
+#### About IP pools
+
+The {{site.prodname}} IPAM block sizes for IP pools are:
+
+- Default: /26 (IPv4)
+- IPv4 range: 20-32, inclusive
+- IPv6 range: 116-128, inclusive
+
+You can have **only one default IP pool for per protocol** in your installation manifest. In this example, there is one IP pool for IPv4 (/26), and one IP pool for IPv6 (/116).
+
+```
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+  pec:
+   # Configures Calico networking.
+   calicoNetwork:
+     # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: 10.48.0.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+    - blockSize: 116
+      cidr: 10.50.0.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+``` 
+
+However, the following is invalid because it has two IP pools for IPv4. 
+
+```
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+  spec:
+   # Configures Calico networking.
+   calicoNetwork:
+     # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: 10.48.0.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+    - blockSize: 31
+      cidr: 10.48.8.0/21
+      encapsulation: IPIP
+      natOutgoing: Enabled
+      nodeSelector: all()
+
+```
+
 #### Expand or shrink IP pool block sizes
 
-By default, the {{site.prodname}} IPAM block size for an IP pool is /26. To expand from the default size /26, lower the `blockSize` (for example, /24). To shrink the `blockSize` from the default /26, raise the number (for example, /28). 
+- To expand a `blockSize`, lower the size. For example, if /26, lower to /24.
```

Comment:
I think I prefer the previous wording - what is the new text trying to improve?

Reply:
Reverted